android/tangram/build.gradle

```diff
@@ -14,44 +14,65 @@ version = VERSION_NAME
 apply from: 'versioning.gradle'
 
 android {
-  compileSdkVersion 24
-
-  /*
-   * Ubuntu can't run the aapt on 64 bit before installing this packages
-   *    sudo apt-get install lib32stdc++6
-   *    sudo apt-get install lib32z1
-   */
-
-  buildToolsVersion '24.0.2'
+  compileSdkVersion 25
+  buildToolsVersion '25.0.0'
 
   defaultConfig {
     minSdkVersion 15
-    targetSdkVersion 24
+    targetSdkVersion 25
     versionCode buildVersionCode()
     versionName VERSION_NAME
+    externalNativeBuild {
+      cmake {
+        targets 'tangram'
+        arguments '-DPLATFORM_TARGET=android',
+                  '-DANDROID_TOOLCHAIN=clang',
```

Comment:
`clang` became the default very recently - I would prefer to leave this here in case someone is using an NDK from before that change.